routes/api/v1.php

```diff
@@ -17,4 +17,11 @@
             ->group(function (Router $router) {
                 $router->get('/', [ReparationRequestController::class, 'index'])->name('index');
             });
+
+        $router
+            ->prefix('reparation_requests_material')
```

Comment:
I think the convention is usually to use the plurar form. I've used it too. For example: https://stripe.com/docs/api/terminal/locations/update